libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentCongestion.kt

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+class IncidentCongestion(
```

Comment:
Do we need an `IncidentCongestion` class to only hold one value? See https://github.com/mapbox/mapbox-navigation-android/pull/3612#discussion_r498480738

BTW if we want that, shouldn't be `private constructor` and a `Builder` needed?

Reply:
Builder is really useful when we have 5+ constructors, this class has only one, and potentially could have 2(+ description).

Reply:
I agree that if there will only be one value, let's move it straight to the alert class.

Reply:
there's should be `description` as well, but for now it's not presented in API. So, let's postpone to include that. We still don't know what possible options of description might be

Reply:
Then we should follow the `Builder` approach.

Reply:
done